Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml

```diff
@@ -88,3 +88,18 @@
     size: 10
     sprite: Objects/Weapons/Melee/combat_knife.rsi
     prefix: inhand
+
+- type: entity
+  name: survival knife
+  parent: CombatKnife
+  id: SurvivalKnife
+  description: Weapon of first and last resort for combatting space carp.
```

Comment:
Combating is one T

Reply:
American linguistic imperialism smdh

Reply:
> American linguistic imperialism smdh

Fair enough, googling gave me my spelling, usually it would give me both.

Reply:
I'm leaning on leaving as is, sorry Americans.